config.toml

```diff
@@ -0,0 +1,8 @@
+main_backend = "main"
+
+[[backend]]
+  type = "vault-kv"
+  name = "main"
+  address = "127.0.0.1:8200"
+  mount = "secret"
+  token = "s.XxzTITDlzOsNNzBlYRHURna6"
```

Comment:
The default config file shouldn't have any specific token. I think a placeholder like `"<vault token>"` would do.

Reply:
Done